json/isa_rv64zihintpause.json

```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "mnemonic" : "pause",
+    "tags" : ["zihintpause"],
+    "overlay" : {
+      "base" : "fence",
+      "match" : ["0xFFFF8F80", "0x01000000"]
```

Comment:
I had to look this up to remember how this works.... so the first value is the "mask" and the second value is the value of the opcode after the mask is applied. I'm guessing the mask here are the fields of the `fence` instruction that make it a `pause`, right? 
```
PAUSE is encoded as a FENCE instruction with pred=W, succ=0, fm=0, rd=x0, and rs1=x0.
```
It'd be nice if we could document this, but I don't know where it would make sense to do that since we can't put comments in JSON files.

Reply:
Yes, based on looking at other instructions that have an overlay, I kinda interpreted the first value of the match as "the bits of the instruction that usually are allowed to vary but must be set a certain way for this instruction". That mask is `AND`d with the instruction, and if the result matches the second value, it is a `pause`.

Agreed that it would be nice to document, but I don't know where either.